SUMMARY.md

```diff
@@ -151,6 +151,8 @@
       * [Add custom styles](guides/plugins/plugins/administration/add-custom-styles.md)
       * [Using the data handling](guides/plugins/plugins/administration/using-data-handling.md)
       * [Using the data grid component](guides/plugins/plugins/administration/using-the-data-grid-component.md)
+      * [Using Utils](guides/plugins/plugins/administration/using-utils.md)
+      * [The Shopware object](guides/plugins/plugins/administration/the-shopware-object.md)
```

Comment:
the file `the-shopware-object.md` is missing in this pr

Reply:
It was indeed available in the `master`, but not yet linked.

Reply:
check